src/transform/classify.rs

```diff
@@ -0,0 +1,64 @@
+/// Transforms for classifying (tagging and categorizing) events.
+///
+/// Based on code in aw_research: https://github.com/ActivityWatch/aw-research/blob/master/aw_research/classify.py
+
+use regex::Regex;
+use serde_json;
+
+use std::collections::HashSet;
+use crate::models::Event;
+
+pub fn classify(events: Vec<Event>, classes: Vec<(String, Regex)>) -> Vec<Event> {
+    // TODO: There is probably a better way that avoids the clone?
```

Comment:
iter() doesn't take ownership of Event and classify_one takes in a mut event so it's not possible currently.
I would suggest to instead of using iter use some iteration technique which takes ownership of the event and actually consumes the vector (drain for example does this).

Reply:
Nice, fixed.